### ISSUE TITLE
build a front-end

### ISSUE THREAD

**Issue by flamingveggies** [hackoregon/emergency-response#17]
build one

**Comment by hannahkane** [comment]
blocked by #13, #14, #15, and #16 

+ we need to figure out how we're going to work with the folks in the Monday/Wednesday class

**Comment by stellarluminary** [comment]
Do we have any updates on that front? The channel's been pretty quiet and it looks like we're still waiting for the data team to solidify things?

**Comment by meganmckissack** [comment]
@lunarnightlight, you might check in about a map component but there's no wireframe in place yet

**Comment by stellarluminary** [comment]
@meganmckissack David's leading the Data Viz class through developing a map component this week. 

I just ask because it looks like only @flamingveggies is able to regularly attend the ER meetings; the junior devs at least are either in React or Data Viz, which meet Mondays and Wednesdays.

**Comment by meganmckissack** [comment]
@lunarnightlight got it.  We'll need a bit of direction from a design point I think first but we can also communicate on Tuesdays at front end build sessions.

**Comment by stellarluminary** [comment]
@meganmckissack You know, that's actually a good idea! I was wondering if the ER front end team would need our own party meetup. Or start a group discussion on the HackUniversity Slack channel since most of us will be over there instead of the ER channel (at least for now).

**Comment by hannahkane** [comment]
+1 to adding a second night for front-end if we need to; though I think we're at least a week or two away from being ready for that

**Comment by stellarluminary** [comment]
@hannahkane Yeah, I'm not too worried about being left out of the loop right now because it looks like we're just not there yet. I mean, almost all of our project issues have to do with the data team.

**Comment by hannahkane** [comment]
@lunarnightlight - yup, plus we'll need a bit of time to figure out some product and UX stuff, too, to give the front-end work some direction. Wireframes coming soon!

**Comment by stellarluminary** [comment]
@hannahkane It's all good. This is just par for the course for any data project.
